### ISSUE TITLE
Grid class in cython

### ISSUE THREAD

**Issue by tberlok** [nbia-astro/skeletor#65]
At the moment our python grid class cannot be passed to cython functions.
For instance, in the cython push we have the following:
```
def boris_push(particle_t[:] particles, real2_t[:, :] E, real_t bz,
               real_t qtmh, real_t dt, int noff, int lbx, int lby):
```
which we would ideally like to replace with
```
def boris_push(particle_t[:] particles, real2_t[:, :] E, real_t bz,
               real_t qtmh, real_t dt, grid_t grid):
```

**Comment by tobson** [comment]
This is now implemented, see b97e2499b7e03736eb7350c0888c2f7beec5437d. The change mentioned in the previous comment has been made in 54bb9d9ebccf563e1da75c71c88b85637c6b85be. I'm leaving this issue open for now since I haven't had time to make analogous changes in other parts of the code.

**Comment by tberlok** [comment]
I think we can close this now. Will you do this?
